internal/test/integration/test_python_elasticsearchclient.go

```diff
@@ -21,28 +21,38 @@ import (
 	ti "go.opentelemetry.io/obi/pkg/test/integration"
 )
 
-func testPythonElasticsearch(t *testing.T) {
-	url := "http://localhost:8381"
-	comm := "python3.12"
-	index := "test_index"
+const (
+	comm          = "python3.12"
+	testIndex     = "test_index"
+	testServerURL = "http://localhost:8381"
+)
 
-	waitForTestComponentsRoute(t, url, "/health")
-	testElasticsearchSearch(t, comm, url, index)
+func testPythonElasticsearch(t *testing.T, dbSystemName string) {
+	var url string
+	switch dbSystemName {
+	case "elasticsearch":
+		url = "http://elasticsearchserver:9200"
+	case "opensearch":
+		url = "http://opensearchserver:9200"
```

Comment:
should this be `9201`?

Reply:
This is the container port so it should be 9200